libraries/AdaptiveExpressions/TriggerTrees/Clause.cs

```diff
@@ -41,19 +44,32 @@ internal Clause(IEnumerable<Expression> children)
         {
         }
 
+        /// <summary>
+        /// Gets or sets of the anyBinding dictionary.
```

Comment:
Gets or set the AnyBindings...

Reply:
'Gets or sets' are suggested standard text for a property in Visual Studio.

Reply:
Correct, "Gets or sets" is correct. I have a typo in my comment. I was pointing at the rest of it. The "of" is not needed and the name of the type should be AnyBinding, capitalized.

Reply:
Sure, I changed the sentence. Thanks for the notes.